srcjs/utils.js

```diff
@@ -240,7 +240,12 @@ function mergeSort(list, sortfunc) {
 
 // Escape jQuery selector metacharacters: !"#$%&'()*+,./:;<=>?@[\]^`{|}~
 var $escape = exports.$escape = function(val) {
-  return val.replace(/([!"#$%&'()*+,.\/:;<=>?@\[\\\]^`{|}~])/g, '\\$1');
+  // Sometime val comes empty, and Js throws a 'e.replace() is not a function' error
+  if (val.length)  {
```

Comment:
This'll still throw an error if `val` is `undefined`. I'd either do `if (val)` or `if (typeof val === "string")`

Reply:
done ✅

I've chosen the `typeof` test, as what we tried to correct is `[].replace()`, and `if ([])` will try to do this 🙂